### DISCUSSION TITLE
Branches on Scaler Pro

### DISCUSSION THREAD

**Discussion by markski1** [planetscale/discussion#612] [FAQs] [answered]
Hello, I have a few questions regarding how additional branches work in Scaler Pro.

I currently have a Scaler database which I am quite happy with. This includes, as a standard, 2 production branches and 5 development ones.

Scaler Pro, however, has only 1 production branch included then it's pay as you go, then as far as development branches go, the page says I'll get 1,440 hours, whatever that means.

Where can I find pricing data on these types of branches? And does this mean I won't have a permanent dev branch anymore unless I pay a premium?

**Comment by orware** [comment, marked as answer]
Hello @markski1,

If referring to production branches specifically pricing information for them can be found here:
https://planetscale.com/pricing

If referring to development branches the included 1,440 development branch hours provides you essentially the equivalent of having 2 development branches included for the whole month, but adds some extra flexibility.

This is a recent change mainly done for flexibility purposes since previously Scaler Pro plans came with 2 development branches and then additional ones were $10/mo extra, but this assumes that you would keep them around the whole month.

In our ideal workflow, development branches are only kept around for a short period of time and then after the associated schema changes they contain are merged in, they would be deleted.

In your case it sounds like you would want to keep at least one development branch around permanently so you can treat that branch as using up half of your included development branch hours for a given month.

The remaining hours can then be utilized for other development branches, hopefully short-lived ones, and would be used throughout the month, but because of the flexibility provided by the development branch hours model now available, you can spin up several if needed and the same pool of remaining hours would be used. 

Any development branch hour usage exceeding the included 1,440 for the month would simply be billed at a small hourly rate, which I believe is around $0.013-$0.014 per hour of usage.

Hopefully this explanation is helpful!

**Reply by markski1** [reply]
Thanks a lot for your reply, this clears things up. In our case the "main" development branch is also used for testing - Which sounds like bad practice but for the workflow we have it makes sense. I'm glad to hear we'd be okay with that then.

Another question, if you don't mind: I see it says "deployed across 3 availabiltiy zones". Does that mean the main server is in a zone and the two replicas are in separate zones or does this refer to something else?

**Reply by orware** [reply]
Hello @markski1,

I'm glad that extra information was helpful!

For the 3 availability zones question, that means that on the Scaler Pro plan, your primary + 2 replicas are each located in a separate availability zone within your selected region. Additionally, our VTGate / load balancing layer for your database is also spread out across multiple availability zones too.

Hopefully that confirms the same line of thinking you were already mentioning above!
